packages/timebrush-powerbi/src/TimeBrushVisual.ts

```diff
@@ -251,6 +252,18 @@ export default class TimeBrush extends StatefulVisual<TimeBrushState> {
     }
 
     /**
+     * Loads the legend
+     */
+    private loadLegendFromPowerBI(state: TimeBrushState) {
+        this.timeBrush.legendFontSize = state.legendFontSize;
+        if (state.showLegend) {
+            this.timeBrush.legendItems =  state.seriesColors;
+        } else {
+            this.timeBrush.legendItems =  [];
+        }
+    }
```

Comment:
This could be a ternary: `this.timeBrush.legendItems = state.showLegend ? state.seriesColors : []; `